README.md

```diff
@@ -118,7 +146,7 @@ You can run a test using your container engine's flags by setting `CONTAINER_ENG
 # make test CONTAINER_ENGINE_BUILD_FLAGS="--build-arg http_proxy=http://yourproxy/" CONTAINER_ENGINE_RUN_FLAGS="-e http_proxy=http://yourproxy/"
 ```
 
-### Dependencies Management
+### Go Dependencies Management
```

Comment:
I'd lean towards not changing this. If at some point we have non-go dependencies (we do have a small C dep, nolibc), we want to mention them in the README and we want a different section, we can do this change. But I'm unsure what it adds now.

Reply:
Because this section talks only about go dependencies. All of the other are handled by package management further up (or go itself, which is separate). To have this section be called "dependencies management", but then have all of the dependencies already listed and this section only discusses go, can be confusing.

Reply:
Ohhh, okay, I completely missed that! I see the point now, sorry!